res/xml/qwerty_vi.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<keyboard>
+  <row>
+    <key key0="q" key1="~" key2="1" key4="esc"/>
+    <key key0="w" key1="loc accent_breve" key2="2" key3="\@" key4="loc accent_horn"/>
+    <key key0="e" key1="!" key2="3" key3="\#" key4="ê"/>
+    <key key0="r" key2="4" key3="$" key1="loc accent_hook_above"/>
+    <key key0="t" key2="5" key3="%"/>
+    <key key0="y" key2="6" key3="^"/>
+    <key key0="u" key2="7" key3="&amp;"/>
+    <key key0="i" key2="8" key3="*"/>
+    <key key0="o" key1="ô" key2="9" key3="(" key4=")"/>
+    <key key0="p" key2="0" key3="f11_placeholder" key4="f12_placeholder"/>
+  </row>
+  <row>
+    <key shift="0.5" key0="a" key1="tab" key2="`" key3="â"/>
+    <key key0="s" key1="loc accent_aigu"/>
+    <key key0="d" key1="loc accent_bar" key2="₫" key3="đ"/>
+    <key key0="f" key1="loc accent_grave"/>
+    <key key0="g" key2="-" key3="_"/>
+    <key key0="h" key2="=" key3="+"/>
+    <key key0="j" key1="loc accent_dot_below" key3="{" key4="}"/>
+    <key key0="k" key1="loc accent_hook" key3="[" key4="]"/>
```

Comment:
There's no `accent_hook`.

Reply:
i forgor 💀
btw, removed in latest commit